src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/NETCoreSdkVersionProperty.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.ComponentModel.Composition;
+using System.Threading.Tasks;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.Tree.Dependencies
+{
+    [Export(typeof(INETCoreSdkVersionProperty))]
```

Comment:
We mock properties in lots of places (see ProjectPropertiesFactory.Create usage), I don't think you need this indirection.

Reply:
we can't mock general properties

Reply:
specifially `ProjectProperties` can be passed a project from which it pull project specific properties.  Unfortunately for this I need to test properties that are created by the dotnet SDK.

Reply:
See src\Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests\ProjectSystem\VS\Automation\VsLangProjectPropertiesTests.cs - OutputType.